src/main.d

```diff
@@ -449,10 +450,20 @@ int main(string[] args)
 	}
 	
 	if (!performSyncOK) {
-		writeln("\n--synchronize or --monitor missing from your command options or use --help for further assistance\n");
-		writeln("No OneDrive sync will be performed without either of these two arguments being present\n");
-		oneDrive.http.shutdown();
-		return EXIT_FAILURE;
+		// was the application just authorised?
+		if (cfg.getValueBool("applicationAuthorizeResponseUri")) {
+			// Application was just authorised
+			log.log("\nApplication has been successfully authorised, however no additional command switches were provided.\n");
+			log.log("Please use --help for further assistance in regards to running this application.\n");
+			oneDrive.http.shutdown();
+			return EXIT_SUCCESS;
+		} else {
+			// Application was not just authorised
+			log.log("\n--synchronize or --monitor switches missing from your command line input. Please add one (not both) of these switches to your command line or use --help for further assistance.\n");
+			log.log("No OneDrive sync will be performed one of these two arguments being present.\n");
```

Comment:
```suggestion
			log.log("No OneDrive sync will be performed without one of these two arguments being present.\n");
```

Reply:
@mussorgsky 
Thanks for the review / pickup